src/aks-preview/azext_aks_preview/addonconfiguration.py

```diff
@@ -0,0 +1,900 @@
+# --------------------------------------------------------------------------------------------

```

Comment:
I suppose this file is copied from some where?

I would suggest break out one new PR for the file renaming or function name renaming.
that will helps a lot for the code reviewers.